Makefile

```diff
@@ -75,6 +75,18 @@ else # GTEST case
   SRCS := $(shell find $(SRC_DIRS) -type f \( -name '*.cpp' -o -name '*.c' \) | grep -v "$(EXCLUDE_DIRS)")
 endif
 
+# Check if BUILD_WEAK_STUBS_SRC is defined
```

Comment:
you need a script now to trigger and automatically call the tests..

And checking whether all the tests pass or not.

Reply:
Have added ut_test_scripts.sh which will check this scenario.